### PR TITLE
Improve feature list + release Beta 2

### PR DESCRIPTION
**Pull request by ildyria**

### CODE REVIEW

File: src/content/post/2024-10-14-v6-beta3.mdx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  - v6
+---
+
+I got some nice and extensive feedback from Beta 1.
+See below the fixes and improvements that have been made in the Beta 2.
+
+- [x] : Fix pictures not being deleted when clicked on the button.
+- [x] : Fix links and bold which were invisible in light mode in Album and Photo descriptions
+- [x] : Add link shape check on Upload via Url
+- [x] : Dropbox upload is now set to `disabled` if it was empty, no more useless warning in Diagnostics.
```
Comment (d7415) [inline src/content/post/2024-10-14-v6-beta3.mdx:20]: :+1:

File: src/content/post/2024-10-14-v6-beta3.mdx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- [x] : Some labels have been updated for readability.
+- [x] : Added a button in the middle of the screen when the album is empty to prompt the upload of pictures.
+- [x] : Drag & Drop fixed on Upload + Full drag & drop support on album pages + Paste image support as upload on album pages.
+- [x] : Ongoing progress of uploads is now displayed on top of the modal.
+- [x] : Added tooltips on the action bar for photos (star, rotate, delete)
+- [x] : In slideshow the actions buttons for photos were still visible (though unusable).
+- [x] : Fixed the Edit photo dialog for consistent design.
+- [x] : Improved visibility form in Albums.
+- [x] : Add set as album header button in photo view
+- [x] : Rotating a photo is refreshes the photo.
```
Comment (d7415) [inline src/content/post/2024-10-14-v6-beta3.mdx:31]:
```suggestion
- [x] : Rotating a photo refreshes the photo.
```

File: src/content/post/2024-10-14-v6-beta3.mdx [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+- [x] : Added setting to preserve photo on upload when auto rotation is set.
+- [x] : Added swipe left / right to switch picture.
+- [x] : Added scroll up / down to switch picture.
+- [x] : Fixed some Warnings as Info in Diagnostics.
+- [x] : Fixed broken images on thumbs
+- [x] : After upload is completed, refresh album in the background
+- [x] : Switching from dark mode to light mode and vice-versa no longer requires reloading.
+- [x] : Fixed Photo details colors in light mode.
+- [x] : Do not show the Map when Map settings is disabled.
+- [x] : Fixes error messages showing up full screen when `dist/user.css` and `dist/custom.js` do not exists.
```
Comment (d7415) [inline src/content/post/2024-10-14-v6-beta3.mdx:41]:
```suggestion
- [x] : Fixes error messages showing up full screen when `dist/user.css` and `dist/custom.js` do not exist.
```

File: src/content/post/2024-10-14-v6-beta3.mdx [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+- [x] : Add `GET api/v2/Photo::random` end point to request random pictures, this provides the full data of the photo instead of the src links for the frame.
+- [x] : Revamped completely the User Management panel.
+
+With this out of the pipe, it is now time to release a [new Beta version](https://github.com/LycheeOrg/Lychee/releases/tag/v6.0.0-beta.2).
+And as always, the Docker `alpha` tag reflect the latest changes to Version 6.
```
Comment (d7415) [inline src/content/post/2024-10-14-v6-beta3.mdx:46]:
```suggestion
And as always, the Docker `alpha` tag reflects the latest changes to Version 6.
```

File: src/pages/[...blog]/[...page].astro [hunk 5]
```diff
@@ -48,5 +49,23 @@ const metadata = {
       <PostTags tags={allCategories} class="mb-2" title="Search by Categories:" isCategory />
       <PostTags tags={allTags}  title="Search by Tags:" />
     -->
-  </section>
+    <CallToAction2
+    classes={{headline: 'bg-white'}}
+    title="Support us!"
+    subtitle="If you are using Lychee, a small token of gratitude will go a long way. You can support further development of Lychee on opencollective or on GitHub."
```
Comment (d7415) [inline src/pages/[...blog]/[...page].astro:55]: As it's hidden.

```suggestion
    subtitle="If you are using Lychee, a small token of gratitude will go a long way. You can support further development of Lychee on GitHub."
```

File: src/pages/[...blog]/index.astro [hunk 6]
```diff
@@ -55,11 +55,11 @@ const metadata = merge(
   title="Support us!"
   subtitle="If you are using Lychee, a small token of gratitude will go a long way. You can support further development of Lychee on opencollective or on GitHub."
```
Comment (d7415) [inline src/pages/[...blog]/index.astro:56]: As above
```suggestion
  subtitle="If you are using Lychee, a small token of gratitude will go a long way. You can support further development of Lychee on GitHub."
```

File: src/pages/support.astro [hunk 7]
```diff
@@ -99,11 +99,11 @@ const metadata = {
     title="Support us!"
     subtitle="If you are using Lychee, a small token of gratitude will go a long way. You can support further development of Lychee on opencollective or on GitHub."
```
Comment (d7415) [inline src/pages/support.astro:100]:
```suggestion
    subtitle="If you are using Lychee, a small token of gratitude will go a long way. You can support further development of Lychee on GitHub."
```

**Review by d7415 — APPROVED** [review]

File: src/content/post/2024-10-14-v6-beta3.mdx (hunk 1 above)
Comment (ildyria) [reply src/content/post/2024-10-14-v6-beta3.mdx:20]: :)

**Pull request merged**